profiles/car.lua

```diff
@@ -62,19 +62,22 @@ local profile = {
     'motor_vehicle',
     'vehicle',
     'permissive',
-    'designated',
-    'destination',
-    'hov' -- we might filter hov out later depending on the avoid settings or add penalties
+    'designated'
   },
 
   access_tag_blacklist = Set {
     'no',
-    'private',
     'agricultural',
     'forestry',
     'emergency',
-    'psv',
-    'delivery'
+    'psv'
```

Comment:
Is there a reason for having the blacklist at all now? If we never route onto the local access ways we're fine in the sense of "blacklisting" them from routing. But if users somehow end up on those ways we still want to be able to route them off of them?